app.js

```diff
@@ -4,26 +4,49 @@ const favicon = require('serve-favicon');
 const logger = require('morgan');
 const cookieParser = require('cookie-parser');
 const bodyParser = require('body-parser');
+const mongoose = require('mongoose');
+const passport = require('passport');
+const localStrategy = require('passport-local').Strategy;
 
-const routes = require('../routes/index');
-const users = require('../routes/users');
+
+const routes = require('./routes/index');
```

Comment:
Can we rename `routes` here?  I always find this initial naming convention from express-generator confusing...